loadgen/cmd/otelbench/main.go

```diff
@@ -98,6 +98,7 @@ func runBench(signal, exporter string, concurrency int) testing.BenchmarkResult
 
 		err := RunCollector(context.Background(), stop, configs(exporter, signal, b.N, concurrency), logsDone, metricsDone, tracesDone)
 		if err != nil {
+			fmt.Println(err.Error())
 			b.Fatal(err)
```

Comment:
```suggestion
			b.Fatal(err.Error())
```

Reply:
This unfortunately does not work. It is written in the description.

Reply:
Whats the output for `./otelbench -test.v ...`? Did you try to redirect output via `-test.testlogfile`?

Reply:
> Whats the output for ./otelbench -test.v ...?

It is:

```
$ go build -o otelbench . && ./otelbench --config=my-config.yaml 
logs-otlp-1                    0                       NaN ns/op
logs-otlphttp-1                0                       NaN ns/op
metrics-otlp-1                 0                       NaN ns/op
metrics-otlphttp-1             0                       NaN ns/op
traces-otlp-1                  0                       NaN ns/op
traces-otlphttp-1              0                       NaN ns/op
```

So no error is printed.

Reply:
>  Did you try to redirect output via -test.testlogfile?

Still doesn't work.